packages/devextreme/js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -191,13 +191,18 @@ export class ResizingController extends modules.ViewController {
     return 'dxDataGrid-ariaDataGrid';
   }
 
-  private _setAriaLabel(): void {
-    const columnCount = this._columnsController?._columns?.filter(({ visible }) => !!visible).length ?? 0;
-    const totalItemsCount = Math.max(0, this._dataController.totalItemsCount());
-    const widgetAriaLabel = this._getWidgetAriaLabel();
-    const widgetStatusText = messageLocalization
-    // @ts-expect-error Badly typed format method
-      .format(widgetAriaLabel, totalItemsCount, columnCount);
+  private _setAriaLabel(e?: any): void {
+    let widgetStatusText = '';
+
+    if (!e?.operationTypes?.fullReload) {
```

Comment:
I'm not sure if you need to use this flag since it will be true when editing and saving data.